controls/raddiagram/howto/raddiagrams-mvvm.md

```diff
@@ -368,7 +370,56 @@ And finally, we need to set the __RadDiagram GraphSource__ property:
 If you run the application now, the __RadDiagram__ should display the following structure:
 ![raddiagram-howto-mvvm-containers](images/raddiagram-howto-mvvm-containers.png)
 
->tip You can download a runnable project of the demonstrated example from our online SDK repository [here](https://github.com/telerik/xaml-sdk), after navigating to __Diagram/MVVM__.					
+>tip You can download a runnable project of the demonstrated example from our online SDK repository [here](https://github.com/telerik/xaml-sdk), after navigating to __Diagram/MVVM__.	
+
+## Using MVVM to bind SourceConnectorPosition and TargetConnectorPosition properties

```

Comment:
Here we don't need the 'Using MVVM to bind' part because we are already in the MVVM article. Also, the names of the properties are not enough to know what the section describe unless you know what they do.

You can try with something like "Bind the source and target connector positions of the connection"

However, I think both titles are okay.